CoreFoundation/AppServices.subproj/CFUserNotification.c

```diff
@@ -190,10 +190,10 @@ static SInt32 _CFUserNotificationSendRequest(CFAllocatorRef allocator, CFStringR
     
 #if TARGET_OS_OSX
     const char *namebuffer = NOTIFICATION_PORT_NAME_MAC;
-    const nameLen = sizeof(NOTIFICATION_PORT_NAME_MAC);
+    const CFIndex nameLen = sizeof(NOTIFICATION_PORT_NAME_MAC);
```

Comment:
I think we usually use `size_t` for these, actually.

Reply:
Except the arguments they are used in expect CFIndexes, so it makes the most sense in this case. Normally you would be right though.